### DISCUSSION TITLE
Field condition: (last) index

### DISCUSSION THREAD

**Discussion by carstenjaksch** [statamic/cms#11303] [Q&A]
I have a replicator set with a field that works only on the third (and last) set. The replicator can have a maximum of four sets.

Currently I have a description for these limitations, but a condition would be nicer. Is there a way to:

1. Show the field on the third set ...
2. ... but only if it is the last set?

**Comment by edalzell** [comment]
maybe with a custom condition?

**Reply by carstenjaksch** [reply]
Thanks for your answer. That is my conclusion too, but I don't know how. As far as I know the condition has no access to the index of a replicator set?

**Reply by edalzell** [reply]
Should be in the VueX store, have you looked there?

**Reply by carstenjaksch** [reply]
Yeah, I had a look but the store is HUGE. 😆 

I have a custom condition called "isIndex" to begin with:

```js
Statamic.$conditions.add('isIndex', ({ params, store }) => {
    console.log(params, store);
});
```

It is used like that: `custom isIndex:2`

I get the store output in the console, but don't know how to get the replicator set index. My knowledge of Vue is limited.

**Reply by edalzell** [reply]
Ya looking through that is confusing, I agree. Always takes me a long time to find things.
